src/main/java/seedu/address/model/EduTrack.java

```diff
@@ -118,6 +127,14 @@ public boolean hasClass(Class c) {
         return classes.contains(c);
     }
 
+    /**
+     * Removes a class from EduTrack.
+     * The person must exist in EduTrack
```

Comment:
Might have overlook a tiny detail, person should be class instead

Reply:
Thanks for pointing that out! 
I've fixed it 👍